### DISCUSSION TITLE
How to change page text (Sample Sample Sample...)

### DISCUSSION THREAD

**Discussion by chuangtsh** [MrGreensWorkshop/MrGreen-JekyllTheme#96] [How to] [answered]
![Web capture_22-7-2022_223917_127 0 0 1](https://user-images.githubusercontent.com/93872867/180464415-bf7341c6-cbff-410a-9b55-67a42c73a212.jpeg)
The line with five "Sample" is what I mean by saying home heading subtitle(I've underlined it). I went to _data/lang/en.yml, and I found out that I could modify [constants: sample] to change the text. But there are always showing five samples, so what should I do to show only one?

**Comment by MrGreensWorkshop** [comment, marked as answer]
Hi, 

Check *.md files, For index page (home) find index.md file on your repo. 

You will see comment suggest `Please delete below and place your page content here`
delete that line below and add this "#hello world" without quotes. That's all.
Texts are added using markdown syntax. Search for "jekyll markdown" on your favorite search engine.

**Reply by chuangtsh** [reply]
That definitely helped! Thank Youuuuuuuuuuu!

a note for who faces the same issue on about page:
go to tabs/about.md and do the same thing

**Reply by MrGreensWorkshop** [reply]
for all pages same, just search for "Please delete below and place your page content here"
I'm glad I could help.
